### ISSUE TITLE
genkeys: $GOPATH/go.mod exists but should not

### ISSUE THREAD

**Issue by loskiq** [yggdrasil-network/yggdrasil-go#547]
I am trying to generate a strong key, but I get this error:
```
root@a1:~# GOPATH=$PWD go run -tags debug misc/genkeys.go
$GOPATH/go.mod exists but should not
root@a1:~#
```

**Comment by neilalexander** [comment]
Try this:
```
go get github.com/yggdrasil-network/yggdrasil-go
GO111MODULE=off go run -tags debug misc/genkeys.go
```

**Comment by loskiq** [comment]
@neilalexander 
```
root@ru:~# go get github.com/yggdrasil-network/yggdrasil-go
package github.com/yggdrasil-network/yggdrasil-go: no Go files in /root/go/src/github.com/yggdrasil-network/yggdrasil-go
root@ru:~#```

**Comment by neilalexander** [comment]
When v0.3.12 is merged (#610) you will now be able to get `genkeys` as follows:
```
go get github.com/yggdrasil-network/yggdrasil-go/cmd/genkeys
```

**Comment by avaldiviezo075** [comment]
Buenas noches esto me funcionó.

#INSTALACIÓN prometeus-slurm-exportador
yum install git golang-bin make

git clone https://github.com/vpenso/prometheus-slurm-exporter.git
cd prometheus-slurm-exporter
export GOPATH=$(pwd):/usr/share/gocode

cd
go get github.com/prometheus/client_golang/prometheus
go get github.com/sirupsen/logrus
go get gopkg.in/alecthomas/kingpin.v2

cd prometheus-slurm-exporter
mv go.mod go.mod_bk

make test
make build
